core/storage/src/trie/mod.rs

```diff
@@ -181,8 +184,8 @@ pub struct Trie {
 pub type DBChanges = HashMap<Vec<u8>, Option<Vec<u8>>>;
 
 impl Trie {
-    pub fn new(storage: Arc<KeyValueDB>, column: Option<u32>) -> Self {
-        Trie { storage, column, null_node: Trie::empty_root() }
+    pub fn new(storage: Arc<KeyValueDB>) -> Self {
+        Trie { storage, column: COL_STATE, null_node: Trie::empty_root() }
```

Comment:
what if we need trie on a different column?

Reply:
It is WIP, the column that the Trie operates with will be hidden from the trie.